app/assets/stylesheets/mixins/variables.scss

```diff
@@ -3,6 +3,8 @@ $elixir-orange: #f47d20;
 $elixir-grey: #484848;
 $elixir-blue: #005472;
 $elixir-green: #bebf32;
+$elixir-dark-green: #b4bcac;
+$elixir-purple: #6c586c;
 
 /* TeSS colours */
 $bgDefault: $elixir-orange;
```

Comment:
if you change it here I think most places will automatically get the right color